src/diet.ts

```diff
@@ -853,6 +868,14 @@ export function consumeDiet(diet: Diet<Note>, name: DietName): void {
             consumeSafe(countToConsume, menuItem.item, menuItem.additionalValue);
           },
         ],
+        [
+          $item`designer sweatpants`,
+          (countToConsume: number) => {
+            for (let n = 1; n <= countToConsume; n++) {
+              useSkill($skill`Sweat Out Some Booze`);
```

Comment:
will this automatically equip the pants?

Reply:
yes

Reply:
😎